content/articles/data-viz-chartify/index.md

```diff
@@ -0,0 +1,336 @@
+
+```python
+ch = chartify.Chart(blank_labels=True, x_axis_type='categorical')
+ch.set_title("Pokemon EDA using Chartify")
+ch.set_subtitle("Number of Pokemon of Each Type in Each Generation")
+ch.plot.parallel(
+        data_frame=generations_types,
+        categorical_columns='generation',
+        numeric_column='count',
+        color_column='type')
+ch.show()
+```
+Just like the bar charts, the line graph takes the in a data_frame argument for our data. We will also specify the `categorical_column`, `numeric_column`, and `color_column` in the same way as it did in the bar chart. After that, we run the code, and our output is displayed.
+
+![Chartify](/engineering-education/data-viz-chartify/chartifyfive.png)
+*Screenshot by author*
+
+We can reproduce this data in a lollipop chart.  
+
+*A lollipop chart is a variation of a bar chart in which the bar has been replaced with a line with a dot at the end, just like a lollipop.* 
+
+Lollipop plots are great when trying to make comparisons between objects or categories. The lollipop chart also requires the same arguments as the bar and line charts. The `axes.set_xaxis_tick_orientation` allows us to set an orientation.
+
+```python
+ch = chartify.Chart(blank_labels=True, y_axis_type='categorical')
+ch.set_title("Pokemon EDA using Chartify")
+ch.set_subtitle("Number of Pokemon of Each Type in Each Generation using a Lollipop chart")
+ch.plot.lollipop(
+        data_frame=generations_types,
+        categorical_columns=['generation','type'],
+        numeric_column='count',
+        color_column='generation')
+ch.axes.set_xaxis_tick_orientation('horizontal')
+ch.show()
+```
+
+![Chartify](/engineering-education/data-viz-chartify/chartifysix.png)
+*Screenshot by author*
+
+### Pokemon Type Combination
+
+We would plot a heatmap to see the pokemon type combination. Just as we did for the other visualizations, we would create a data frame. 
+
+```Python
+type_combos= (pokemon_df.groupby(['type1', 'type2'])['generation'].mean().reset_index())
+```
+And we would use the `.plot.heatmap` to create the heatmap. 
+
+```python
+ch = chartify.Chart(blank_labels=True, x_axis_type='categorical', y_axis_type='categorical')
+ch.plot.heatmap(
+        data_frame=type_combos,
+        x_column='type1',
+        y_column='type2',
+        color_column='generation',
+        text_column='generation',
+        text_color='white')
+ch.axes.set_xaxis_label('type1')
+ch.axes.set_yaxis_label('type2')
+ch.set_title('Pokemon EDA using Chartify')
+ch.set_subtitle("Pokemon Type Combinations: Heatmap")
+ch.show()
+```
+
+![Chartify](/engineering-education/data-viz-chartify/chartifyseven.png)
+*Screenshot by author*
+
+### Pokemon Weight distribution
+
+Let�s have a look at our pokemon weight. This will best be displayed in a histogram. In the �weight_kg� column, there were nan values. We will have to fill it. For this article, we would fill it with 0.
```

Comment:
```suggestion
Let�s have a look at our pokemon weight. This will best be displayed in a histogram. In the `weight_kg` column, there were nan values. We will have to fill it. For this article, we would fill it with `0`.
```

Reply:
Also use back ticks here